response_operations_ui/templates/layouts/base.html

```diff
@@ -19,7 +20,7 @@
     <link href="/static/images/favicon.ico" rel="Shortcut Icon" />
   </head>
 
-  <body>
+  <body style="position: relative; height: auto; min-height: 100%;">
```

Comment:
Is this right?  I've always been told that inline styling is bad

Reply:
I'll discuss with Nic when he;s back in, this is from his templating